src/ui/map.js

```diff
@@ -192,9 +192,9 @@ class Map extends Camera {
 
         bindHandlers(this, options);
 
-        this._hash = options.hash && (new Hash()).addTo(this);
+        this.hash = options.hash && (new Hash()).addTo(this);
```

Comment:
oooh good point – for some reason I was thinking the convention was no `_` if its called from w/in another module but that is not correct 😊